src/main/kotlin/commands/LmgtfyCommand.kt

```diff
@@ -0,0 +1,29 @@
+package commands
+
+import Command
+import arg
+import doesLater
+import greedyString
+
+object LmgtfyCommand : Command("lmgtfy") {
+    init {
+        greedyString("search term") {
+            doesLater { context ->
+                // Explicit types are necessary for type inference
```

Comment:
unnecessary comment

Reply:
oops, yeah that came from the example command